src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -579,6 +580,21 @@ public bool IsTupleCompatible()
         }
 
         /// <summary>
+        /// Verify if the given type is a tuple of a given cardinality, or can be used to back a tuple type 
+        /// with the given cardinality. 
+        /// </summary>
+        public bool IsTupleOrCompatibleWithTupleOfCardinality(int targetCardinality)
```

Comment:
I am sticking with this name because the meaning of the parameter is very different from IsTupleCompatible

---
In reply to: [62070448](https://github.com/dotnet/roslyn/pull/11047#discussion_r62070448) [](ancestors = 62070448)